package.json

```diff
@@ -130,12 +130,12 @@
     "browser-locale": "^1.0.3",
     "classnames": "^2.2.1",
     "cookies-js": "^1.2.1",
-    "counterpart": "^0.18.5",
+    "counterpart": "^0.17.0",
     "event-emitter": "^0.3.4",
     "file-saver": "^1.3.3",
     "foundation-apps": "git+https://github.com/zurb/foundation-apps.git",
     "fractional": "^1.0.0",
-    "highcharts": "4.2.6",
+    "highcharts": "6.0.7",
```

Comment:
I'm actually not sure why we have highcharts listed as a separate package, it's not actually needed since it's part of react-highcharts.